### PR TITLE
Add tests for docs and move the README to website

### PR DESCRIPTION
**Pull request by yogeshjain999**

### CODE REVIEW

**Comment by apotonick** [comment]
Can we leave some tiny examples in the README, some mouth-watering appetizer for this gem, and then link to the website?

**Comment by yogeshjain999** [comment]
Done bro!! Added some description and very simple examples.

**Review by apotonick — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,249 +30,111 @@ require "trailblazer/test/deprecation/operation/assertions"
...
-
-We are also providing 2 helper methods:
-* `params(new_params)`
-* `ctx(new_params, options)`
-
-Those will merge params and options for you and return the final inputs which then can be passed to the operation under testing.
-
-Pass `deep_merge: false` to the helper methods to disable the default deep merging of params and options.
-
-*Same API for Trailblazer v2.0 and v2.1.*
```
Comment (emaglio) [inline README.md:52]: feels like this stuff should be explained somewhere if that still there really...I haven't been following this gem much...

File: README.md [hunk 2]
```diff
@@ -36,249 +30,111 @@ require "trailblazer/test/deprecation/operation/assertions"
...
+[Learn more](http://2019.trailblazer.to/2.1/docs/trailblazer.html#trailblazer-test-assertions)
 
-Finally, using the built-in assertions you are able to test your operations in a fast and easy way:
-* `assert_pass` -> your operation is successful and model has the correct attributes
-* `assert_fail` -> your operation fails and returns some specific errors
-* `assert_policy_fail` -> your operation fails because policy fails
+#### assert_pass
 
-#### params
```
Comment (emaglio) [inline README.md:59]: same for this and the `ctx` below...

**Review by emaglio — APPROVED** [review]
If Nick has approved this means that you guys spoken about these changes and maybe these docs are moved somewhere else 👍

**Comment by yogeshjain999** [comment]
Thanks for the review @emaglio 🍻 
Detailed overview has been moved and updated into website documentation, it'll be there in next deployment.

File: README.md (hunk 1 above)
Comment (yogeshjain999) [reply README.md:52]: Yep, this is also moved.

**Pull request merged**